Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4233,8 +4531,12 @@ def to_context(self):
             if self.type:
                 account_context['Type'] = self.type
 
-            irrelevent = ['CONTEXT_PATH', 'to_context', 'dbot_score', 'Id']
+            if self.blocked:
+                account_context['Blocked'] = self.blocked
```

Comment:
Why is this the only change in the 'to_context' function of the Account class?
Why aren't new other fields such as 'stix_id', 'tags', 'traffic_light_protocol' added here?

Reply:
Lack of attention, or we decided to ditch Account, can't remember, added it anyway

Reply:
Done